nav2_amcl/src/amcl_node.cpp

```diff
@@ -1215,7 +1215,7 @@ AmclNode::updateParametersCallback(
     map_sub_ = create_subscription<nav_msgs::msg::OccupancyGrid>(
       map_topic_,
       std::bind(&AmclNode::mapReceived, this, std::placeholders::_1),
-      nav2::qos::LatchedSubscriptionQoS());
+      nav2::qos::LatchedSubscriptionQoS(1));
```

Comment:
I think we talked about this at ROSCon - we probably want this to be 2-3 so that if one message is coming in, it is not interrupted / canceled by a new one by the DDS / middleware. Making this ~2-3 allows for current communicated messages to be processed completely while a second one is starting to be received